ecommerce/serializers.py

```diff
@@ -299,3 +299,38 @@ class Meta:
         ]
         model = models.Order
         depth = 1
+
+
+class DiscountSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.Discount
+        fields = [
+            "id",
+            "amount",
+            "automatic",
+            "discount_type",
+            "redemption_type",
+            "max_redemptions",
+            "discount_code",
+            "order_redemptions",
```

Comment:
I'm thinking maybe because `order_redemptions` could be a large set, we should remove this and defer to the nested API which should be paginated.